README.md

```diff
@@ -1,13 +1,14 @@
-chromebrew
+![](https://sajjaadf.github.io/chromebrew-Dev/images/brew.png) <br>
```

Comment:
This url should be referencing this repo, not sajjaadf.github.io.

Reply:
Whoops

Reply:
Actually that is for the image

Reply:
Fine

Reply:
When the new website goes up as the pull request gets accepted, it will worj

Reply:
Sure, it will work but the right way is to pull the image from this repo.

Reply:
fine

Reply:
fixed